src/Formatter/ReflectionPropertyName.php

```diff
@@ -10,10 +10,15 @@ final class ReflectionPropertyName
 {
     public function __invoke(ReflectionProperty $property): string
     {
+        $className = $property->getDeclaringClass()->getName();
```

Comment:
Avoid repeated declaring class calls, if possible

Reply:
I've replaced the repeated `getDeclaringClass` calls by a single call